Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -0,0 +1,79 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class SelectCurrencyViewController: UIViewController, MessageBannerViewControllerPresenting {
+  private let viewModel: SelectCurrencyViewModelType = SelectCurrencyViewModel()
+
+  internal var messageBannerViewController: MessageBannerViewController?
+  private var saveButtonView: LoadingBarButtonItemView!
+
+  internal static func instantiate() -> SelectCurrencyViewController {
+    return SelectCurrencyViewController(nibName: nil, bundle: nil)
+  }
+
+  public func configure(with currency: Currency) {
+    self.viewModel.inputs.configure(with: currency)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.view.addSubview(self.tableView)
+    self.tableView.constrainEdges(to: self.view)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.tableView
+      |> settingsTableViewStyle
+      |> \.separatorStyle .~ .singleLine
+  }
+
+  // MARK: Subviews
+
+  private lazy var tableView: UITableView = {
+    let tableView = UITableView(frame: .zero, style: .plain)
+
+    tableView.translatesAutoresizingMaskIntoConstraints = false
+    tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
```

Comment:
I think this is traditionally done in `viewDidLoad` but I'm kinda seeing the point here...not sure which one I like better 😄

Reply:
How about making the `reuseIdentifier` type safe (by using an enum and more appropriate string `"SelectCurrencyCell"`?

Reply:
> I think this is traditionally done in `viewDidLoad` but I'm kinda seeing the point here...not sure which one I like better 😄

😄 yeah not sure, i tend to like how `lazy var` let's us group configuration together, but also could go either way.

I've moved it to a local constant - not sure an `enum` makes sense with only on case?